base/common/src/test/java/com/netscape/certsrv/system/TPSConnectorDataTest.java

```diff
@@ -33,7 +35,7 @@ public void testJSON() throws Exception {
         String json = before.toJSON();
         System.out.println("JSON (before): " + json);
 
-        TPSConnectorData afterJSON = TPSConnectorData.fromJSON(json);
+        TPSConnectorData afterJSON = JSONSerializer.fromJSON(json, TPSConnectorData.class);
         System.out.println("JSON (after): " + afterJSON.toJSON());
```

Comment:
Similar issue as in the other PR, we're not supposed to use `JaxbAnnotationIntrospector` anymore, but please verify that the new JSON mapping produces the same result as before.

Reply:
Seems OK to me
```
##JacksonAnnotationIntrospector
JSON (before): {
  "id" : "tps0",
  "userID" : "user1"
}
JSON (after): {
  "id" : "tps0",
  "userID" : "user1"
}

##JaxbAnnotationIntrospector
JSON (before): {
  "id":"tps0",
  "UserID":"user1"
}
JSON (after): {
  "id":"tps0",
  "UserID":"user1"
}

Reply:
Rebased onto #3598 so it uses `JacksonAnnotationIntrospector` now